sinch/domains/voice/models/conferences/requests.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass
+from sinch.core.models.base_model import SinchRequestBaseModel
+
+
+@dataclass
+class GetVoiceConferenceRequest(SinchRequestBaseModel):
+    conference_id: str
+
+
+@dataclass
+class KickAllVoiceConferenceRequest(SinchRequestBaseModel):
+    conference_id: str
+
+
+@dataclass
+class ManageParticipantVoiceConferenceRequest(SinchRequestBaseModel):
+    conference_id: str
+    call_id: str
+    command: str
```

Comment:
Can we put litteral here as well? https://developers.sinch.com/docs/voice/api-reference/voice/tag/Conferences/#tag/Conferences/operation/Calling_ManageConferenceParticipant!path=command&t=request

Reply:
There is an enum already for that:
```
class ConferenceCommand(Enum):
    MUTE = "mute"
    UNMUTE = "unmute"
    ONHOLD = "onhold"
    RESUME = "resume"
```